pyiron_atomistics/atomistics/structure/periodic_table.py

```diff
@@ -349,15 +353,12 @@ def add_element(
             self.dataframe = self.dataframe.append(parent_element_data_series)
         else:
             self.dataframe.loc[new_element] = parent_element_data_series
-        if len(qwargs) != 0:
```

Comment:
This is what broke the tests before?

Reply:
Yes but not only this. The recent commit https://github.com/pyiron/pyiron_atomistics/pull/224/commits/b278013a174e40b0d0155afa100ff2b58c4cf495 also helped fix the tests